collect_app/src/main/java/org/odk/collect/android/fragments/DataManagerList.java

```diff
@@ -149,7 +143,7 @@ protected void onResume() {
     }
 
     @Override
-    protected void onPause() {
```

Comment:
`onPause()` is overriden from `Fragment` class with public access modifier. You cannot override a method and reduce visibility. You can only increase the visibility (e.g. overriding a protected method and making it public). Is there any specific reason as to why we need to make this method `protected`?